python/Ganga/testlib/GangaUnitTest.py

```diff
@@ -186,9 +214,7 @@ def stop_ganga():
     ShutdownManager._ganga_run_exitfuncs()
 
     # Undo any manual editing of the config and revert to defaults
-    from Ganga.Utility.Config import allConfigs
-    for package in allConfigs.values():
-        package.revertToDefaultOptions()
+    clear_config()
```

Comment:
It's a special case because in the normal use of Ganga, there's no point in clearing the config on shutdown. It takes time when the process is about to end anyway.